content/learn/book/migration-guides/0.8-0.9/_index.md

```diff
@@ -0,0 +1,638 @@
+let entity = commands.spawn().id();
+// New (0.9)
+let entity = commands.spawn_empty().id();
+
+// Old (0.8)
+let entity = world.spawn().id();
+// New (0.9)
+let entity = world.spawn_empty();
+```
+
+### [Accept Bundles for insert and remove. Deprecate `insert`/`remove_bundle`](https://github.com/bevyengine/bevy/pull/6039)
+
+Replace `insert_bundle` with `insert`:
+
+```rust
+// Old (0.8)
+commands.spawn().insert_bundle(SomeBundle::default());
+// New (0.9)
+commands.spawn().insert(SomeBundle::default());
+```
+
+Replace `remove_bundle` with `remove`:
+
+```rust
+// Old (0.8)
+commands.entity(some_entity).remove_bundle::<SomeBundle>();
+// New (0.9)
+commands.entity(some_entity).remove::<SomeBundle>();
+```
+
+Replace `remove_bundle_intersection` with `remove_intersection`:
+
+```rust
+// Old (0.8)
+world.entity_mut(some_entity).remove_bundle_intersection::<SomeBundle>();
+// New (0.9)
+world.entity_mut(some_entity).remove_intersection::<SomeBundle>();
+```
+
+Consider consolidating as many operations as possible to improve ergonomics and cut down on archetype moves:
+
+```rust
+// Old (0.8)
+commands.spawn()
+  .insert_bundle(SomeBundle::default())
+  .insert(SomeComponent);
+
+// New (0.9) - Option 1
+commands.spawn().insert((
+  SomeBundle::default(),
+  SomeComponent,
+))
+
+// New (0.9) - Option 2
+commands.spawn_bundle((
+  SomeBundle::default(),
+  SomeComponent,
+))
+```
+
+### [Fix inconsistent children removal behavior](https://github.com/bevyengine/bevy/pull/6017)
+
+* Queries with `Changed<Children>` will no longer match entities that had all of their children removed using `remove_children`.
+* `RemovedComponents<Children>` will now contain entities that had all of their children remove using `remove_children`.
+
+### [`Query` filter types must be `ReadOnlyWorldQuery`](https://github.com/bevyengine/bevy/pull/6008)
+
+Query filter (`F`) generics are now bound by `ReadOnlyWorldQuery`, rather than `WorldQuery`. If for some reason you were requesting `Query<&A, &mut B>`, please use `Query<&A, With<B>>` instead.
+
+### [Change UI coordinate system to have origin at top left corner](https://github.com/bevyengine/bevy/pull/6000)
+
+All flex layout should be inverted (ColumnReverse => Column, FlexStart => FlexEnd, WrapReverse => Wrap)
+System where dealing with cursor position should be changed to account for cursor position being based on the top left instead of bottom left
+
+### [Clarify `bevy::ui::Node` field and documentation](https://github.com/bevyengine/bevy/pull/5995)
+
+All references to the old `size` name has been changed, to access `bevy::ui::Node` `size` field use `calculated_size`
+
+### [Remove `AssetServer::watch_for_changes()`](https://github.com/bevyengine/bevy/pull/5968)
+
+`AssetServer::watch_for_changes()` was removed.
+Instead, use the `AssetServerSettings` resource.
+
+```rust
+app // AssetServerSettings must be inserted before adding the AssetPlugin or DefaultPlugins.
+ .insert_resource(AssetServerSettings {

```

Comment:
on line 118 above there's the section about AssetServerSettings resource getting changed to AssetPlugin, but it's still using the old style here.